src/api/app/services/meta_controller_service/project_updater.rb

```diff
@@ -10,13 +10,14 @@ def call
       @validator.call
       unless @validator.valid?
         @errors = @validator.errors
-        return
+        return self
       end
 
       Project.transaction do
         @errors = @project.update_from_xml(@request_data)[:error]
         @project.store if valid?
       end
+      self
```

Comment:
Please add a reason for the change to the commit message

Reply:
done, please check @bgeuken

Reply:
Thanks. There are a few typos you might want to fix:-)

Reply:
done @bgeuken, please review it again